src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.spec.ts

```diff
@@ -1,16 +1,35 @@
+import { Component, Input } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
+
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import { NgxPageScrollCoreModule } from 'ngx-page-scroll-core';
+
+import { TemplateSession } from '../../../../services/feedback-sessions.service';
+import { Course } from '../../../../types/api-output';
+import {
+  SessionEditFormMode,
+  SessionEditFormModel,
+} from '../../../components/session-edit-form/session-edit-form-model';
 import { InstructorHelpSessionsSectionComponent } from './instructor-help-sessions-section.component';
 
+@Component({ selector: 'tm-example-box', template: '' })
+class ExampleBoxStubComponent {}
+@Component({ selector: 'tm-session-edit-form', template: '' })
+class SessionEditFormStubComponent {
+  @Input() formMode: SessionEditFormMode = SessionEditFormMode.ADD;
+  @Input() model: SessionEditFormModel = {} as SessionEditFormModel;
+  @Input() courseCandidates: Course[] = [];
+  @Input() templateSessions: TemplateSession[] = [];
+}
+
 describe('InstructorHelpSessionsSectionComponent', () => {
   let component: InstructorHelpSessionsSectionComponent;
   let fixture: ComponentFixture<InstructorHelpSessionsSectionComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [InstructorHelpSessionsSectionComponent],
+      declarations: [InstructorHelpSessionsSectionComponent, ExampleBoxStubComponent, SessionEditFormStubComponent],
```

Comment:
And declaring `ExampleBoxComponent` should also be fine.

Reply:
Will make the change for this component